### ISSUE TITLE
LightRaysNode.js, Uninstrumented code detected

### ISSUE THREAD

**Issue by phet-steele** [phetsims/scenery-phet#304]
```
Error: Assertion failed: Uninstrumented code detected
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1493742068468/assert/js/assert.js:21:13)
    at Function.indicateUninstrumentedCode (https://bayes.colorado.edu/continuous-testing/snapshot-1493742068468/tandem/js/Tandem.js?bust=1493743905927:321:21)
    at new LightRaysNode (https://bayes.colorado.edu/continuous-testing/snapshot-1493742068468/scenery-phet/js/LightRaysNode.js?bust=1493743905927:30:12)
    at new CustomLightBulbNode (https://bayes.colorado.edu/continuous-testing/snapshot-1493742068468/circuit-construction-kit-common/js/common/view/CustomLightBulbNode.js?bust=1493743905927:75:23)
    at new CCKIcon (https://bayes.colorado.edu/continuous-testing/snapshot-1493742068468/circuit-construction-kit-common/js/common/view/CCKIcon.js?bust=1493743905927:62:25)
    at new IntroScreen (https://bayes.colorado.edu/continuous-testing/snapshot-1493742068468/circuit-construction-kit-dc/js/intro/IntroScreen.js?bust=1493743905927:30:23)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1493742068468/circuit-construction-kit-dc/js/circuit-construction-kit-dc-main.js?bust=1493743905927:53:9
    at window.phetLaunchSimulation (https://bayes.colorado.edu/continuous-testing/snapshot-1493742068468/joist/js/SimLauncher.js?bust=1493743905927:50:11)
    at doneLoadingImages (https://bayes.colorado.edu/continuous-testing/snapshot-1493742068468/joist/js/SimLauncher.js?bust=1493743905927:60:18)
    at Object.launch (https://bayes.colorado.edu/continuous-testing/snapshot-1493742068468/joist/js/SimLauncher.js?bust=1493743905927:114:9)
Approximately 5/2/2017, 10:21:08 AM
```

**Comment by zepumph** [comment]
@samreid I think you fixed this in your commits yesterday, would you please confirm?

**Comment by samreid** [comment]
Correct, I fixed this in cc034db95ec0ee011f0cc8b510b6e3f4234ea0f3, closing.
